packages/MainUI/src/components/navigation.tsx

```diff
@@ -82,6 +82,8 @@ const Navigation: React.FC = () => {
         roles={[]}
         selectedRole={null}
         selectedWarehouse={null}
+        saveAsDefault={false}
+        onSaveAsDefaultChange={() => {}}
```

Comment:
Please avoid declarations in JSX.
In this case, you could declare the "noop" callback outside of this component and reference it here.
```tsx

const noop = () => {};
export const MyCmp = (props) => <TheChild onSaveAsDefaultChange={noop}>...</>
```